src/cclib/io/filewriter.py

```diff
@@ -89,6 +92,13 @@ def _make_bond_connectivity_from_openbabel(self, obmol):
                                         obbond.GetBondOrder()))
         return bond_connectivities
 
+    def _check_required_attributes(self, required):
+        """Check if required attributes are present in ccdata"""
+        missing = [x for x in required if not hasattr(self.ccdata, x)]
+        if len(missing) > 0:
+            missing = ' '.join(missing)
```

Comment:
It's bad practive to re-use the same variable name. I think it's safe to include the joining inline in the next line anyway.